packages/roosterjs-editor-core/lib/corePlugins/utils/InlineEntityHandlers/inlineEntityOnPluginEvent.ts

```diff
@@ -20,9 +24,18 @@ import type { Entity, PluginEvent } from 'roosterjs-editor-types';
 const DELIMITER_SELECTOR =
     '.' + DelimiterClasses.DELIMITER_AFTER + ',.' + DelimiterClasses.DELIMITER_BEFORE;
 const ZERO_WIDTH_SPACE = '\u200B';
+const INLINE_ENTITY_SELECTOR = 'span' + getEntitySelector();
 
 export function inlineEntityOnPluginEvent(event: PluginEvent, editor: IEditor) {
     switch (event.eventType) {
+        case PluginEventType.EditorReady:
```

Comment:
Also need this for ContentChangedEvent when change source == ChangeSource.SetContent